packages/fluent/scss/stepper/_layout.scss

```diff
@@ -0,0 +1,291 @@
+            display: flex;
+            align-items: center;
+            overflow: hidden;
+        }
+
+        // Step indicator
+        .k-step-indicator {
+            @include border-radius( $kendo-stepper-indicator-border-radius );
+            margin: if( $kendo-stepper-indicator-focus-offset > 0, $kendo-stepper-indicator-focus-offset, null );
+            width: var( --kendo-stepper-indicator-width, #{$kendo-stepper-indicator-width} );
+            height: var( --kendo-stepper-indicator-height, #{$kendo-stepper-indicator-height} );
+            border-width: var( --kendo-stepper-indicator-border-width, #{$kendo-stepper-indicator-border-width} );
+            border-style: solid;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex: none;
+            position: relative;
+            z-index: 1;
+            overflow: visible;
+            transition-property: color, background-color, border-color;
+            transition-duration: .4s;
+            transition-timing-function: ease-in-out;
+
+            &::after {
+                @include border-radius( 100% );
+                content: "";
+                border-width: var( --kendo-stepper-indicator-focus-border-width, #{$kendo-stepper-indicator-focus-border-width} );
+                border-style: solid;
+                border-color: inherit;
+                pointer-events: none;
+                display: none;
+                position: absolute;
+                top: calc( -1 * #{$kendo-stepper-indicator-focus-calc-offset} );
+                right: calc( -1 * #{$kendo-stepper-indicator-focus-calc-offset} );
+                bottom: calc( -1 * #{$kendo-stepper-indicator-focus-calc-offset} );
+                left: calc( -1 * #{$kendo-stepper-indicator-focus-calc-offset} );
+                z-index: 2;
+            }
+        }
+
+        .k-step.k-focus,
+        .k-step-link:focus {
+            .k-step-indicator::after {
+                display: block;
+            }
+        }
+
+        .k-step.k-disabled {
+            .k-step-indicator::after {
+                display: none;
+            }
+        }
+
+
+        // Step label
+        .k-step-label {
+            max-width: clamp(100%, 10em, 100%);
+            display: inline-flex;
+            flex-wrap: wrap;
+            align-items: center;
+            justify-content: center;
+            z-index: 1;
+        }
+        .k-step-label:only-child {
+            @include border-radius( calc(($kendo-stepper-indicator-height + 2 * $kendo-stepper-indicator-border-width) / 2) );
+            padding-inline: var( --kendo-stepper-label-padding-x, #{$kendo-stepper-label-padding-x} );
+            padding-block: var( --kendo-stepper-label-padding-y, #{$kendo-stepper-label-padding-y} );
+            border-width: 0;
+        }
+        .k-step-label .k-step-text {
+            max-width: calc(10em - (#{$kendo-icon-size} + #{$kendo-icon-spacing}));
```

Comment:
for icon size and icon spacing you can use the CSS vars that are exposed in the root